moped-database/migrations/1690917480404_proj_funding_updates/up.sql

```diff
@@ -0,0 +1,26 @@
+SELECT
+    setval('moped_fund_source_cat_funding_source_category_id_seq', (
+            SELECT
+                MAX(funding_program_id)
+                FROM moped_fund_programs));
```

Comment:
at least locally, this sequence was out of whack when i tried to insert new records. this resets it.

Reply:
Thanks for sorting this out. I couldn't recall if sequences store the next value or the current value, so I threw together this tiny test case, and it this is perfect. 👍 

```sql
CREATE TABLE frank_test (
    id SERIAL PRIMARY KEY,
    data INTEGER
);

insert into frank_test (data) values (1);
select * from frank_test;

SELECT currval('frank_test_id_seq');

insert into frank_test (data) values (1);
select * from frank_test;

SELECT currval('frank_test_id_seq');
```